qsirecon/tests/test_cli.py

```diff
@@ -16,7 +16,16 @@
     download_test_data,
     get_test_data_path,
 )
-from qsirecon.utils.bids import write_derivative_description
+
+
+from qsirecon.utils.bids import (
+    write_derivative_description,
+    write_bidsignore,
+    write_atlas_dataset_description,
+)
+from qsirecon.cli.workflow import copy_boilerplate
+
+from .utils import freesurfer_license
```

Comment:
Probably should consistently use absolute or relative imports.

Reply:
BTW I'd love to move the test folder out of the qsirecon folder like this:

```
src/
     qsirecon/
tests/
```

I don't know if that's possible, but it would be awesome in terms of code cleanliness IMHO. If it _is_ possible, then tests should definitely have absolute imports.